### PR TITLE
Update to openjdk8 to openjdk8-openj9 for the scala base image to dea…

### PR DESCRIPTION
**Pull request by somaya10**
…l with vulnerabilities

## Description
Update to openjdk8 to openjdk8-openj9 for the scala base image to deal with vulnerabilities
Updating the image in the Dockerfile will also update the Controller and Invoker.

openjdk8-openj9:alpine version causes no regression in Latency as compared to openjdk8:x86_64-alpine-jdk8u202-b08(previously opened PR https://github.com/apache/incubator-openwhisk/pull/4353).

openjdk8-openj9:alpine
Screen Shot 2019-04-11 at 14 33 03

openjdk8:x86_64-alpine-jdk8u202-b08
Screen Shot 2019-04-09 at 17 29 46

## Related issue and scope

- [ ] I opened an issue to propose and discuss this change (#????)

## My changes affect the following components

- [ ] API
- [x] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [x] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [ ] Tests
- [ ] Deployment
- [ ] CLI
- [ ] General tooling
- [ ] Documentation

## Types of changes

- [ ] Bug fix (generally a non-breaking change which closes an issue).
- [x] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [x] I signed an [Apache CLA](https://github.com/apache/incubator-openwhisk/blob/master/CONTRIBUTING.md).
- [x] I reviewed the [style guides](https://github.com/apache/incubator-openwhisk/wiki/Contributing:-Git-guidelines#code-readiness) and followed the recommendations (Travis CI will check :).
- [ ] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [ ] I updated the documentation where necessary.

### CODE REVIEW

**Comment by somaya10** [comment]
PG2-4376

**Comment by dgrove-oss** [comment]
Could please summarize the performance impact if any of making this change in the PR.

File: common/scala/Dockerfile [hunk 1]
```diff
@@ -1,15 +1,15 @@
 # Licensed to the Apache Software Foundation (ASF) under one or more contributor
 # license agreements; and to You under the Apache License, Version 2.0.
 
-FROM adoptopenjdk/openjdk8:x86_64-alpine-jdk8u172-b11
+FROM adoptopenjdk/openjdk8-openj9:alpine
```
Comment (dubee) [inline common/scala/Dockerfile]: Can you performance test `FROM openjdk:8u201-jdk-alpine3.9` since the JDK has memory issue?

Comment (somaya10) [reply common/scala/Dockerfile]: @dubee The performance doesnt look promising with this version.

Screen Shot 2019-04-15 at 10 14 35

**Comment by dgrove-oss** [comment]
Closing as outdated.

**Pull request closed without merging**